draft-aegis-mimi-arch-latest.md

```diff
@@ -301,7 +369,7 @@ Output:
 
 In the above figure, the returned `clientId` is the new unique
 identifier assigned to the client by the identity service. Services MUST ensure
-that `clientIdentity` is unique amongst all registered clients. 
+that `ClientID` is unique amongst all registered clients. 
```

Comment:
The previous sentence already enforces that `clientId` is unique. This is an additional requirement on the handle.

Reply:
Dropped the word unique in first sentence and left second sentence. (The second sentence is explicit that its the services job to ensure uniqueness.)